internal/tfengine/tfengine.go

```diff
@@ -33,8 +33,9 @@ import (
 
 // Options is the options for tfengine execution.
 type Options struct {
-	Format   bool
-	CacheDir string
+	Format    bool
+	CacheDir  string
+	Templates map[string]bool
```

Comment:
Naming this Templates is a bit confusing. Maybe something like WantedTemplates or something? 

Also add a comment above this field to describe its purpose and that a len of 0 means get all templates.

Reply:
Done.